OpenRA.Mods.Common/Traits/Parachutable.cs

```diff
@@ -19,16 +19,20 @@ class ParachutableInfo : ITraitInfo
 		[Desc("If we land on invalid terrain for my actor type should we be killed?")]
 		public readonly bool KilledOnImpassableTerrain = true;
 
+		[Desc("Group where *CorposeSequence is looked up.")]
+		public readonly string CorpseSequenceCollection = "explosion";
+
 		public readonly string GroundImpactSound = null;
-		public readonly string GroundCorpseSequence = "corpse";
+		[SequenceReference("CorpseSequenceCollection")] public readonly string GroundCorpseSequence = "corpse";
 		public readonly string GroundCorpsePalette = "effect";
 
 		public readonly string WaterImpactSound = null;
-		public readonly string WaterCorpseSequence = "small_splash";
+		[SequenceReference("CorpseSequenceCollection")] public readonly string WaterCorpseSequence = null;
```

Comment:
Why are you removing the default value here?
While I don't like such magical default values, this will probably need an upgrade rule, and I really don't want this PR to get any bigger.